tools/README.md

```diff
@@ -1,658 +1,11 @@
-  -f, --filter-criteria  <arg>    Filter newest or oldest N event logs for processing.
-                                  Supported formats are:
-                                  To process 10 recent event logs: --filter-criteria "10-newest-timestamp"
-                                  To process 10 oldest event logs: --filter-criteria "10-oldest-timestamp"
-  -g, --generate-dot              Generate query visualizations in DOT format.
-                                  Default is false
-  -m, --match-event-logs  <arg>   Filter event logs filenames which contains the input string.
-  -n, --num-output-rows  <arg>    Number of output rows for each Application.
-                                  Default is 1000
-  -o, --output-directory  <arg>   Base output directory. Default is current
-                                  directory for the default filesystem. The
-                                  final output will go into a subdirectory
-                                  called rapids_4_spark_profile.
-                                  It will overwrite any existing files with
-                                  the same name.
-  -p, --print-plans               Print the SQL plans to a file starting with
-                                  'planDescriptions-'. Default is false
-  -h, --help                      Show help message
-
- trailing arguments:
-  eventlog (required)   Event log filenames(space separated) or directories
-                        containing event logs. eg: s3a://<BUCKET>/eventlog1
-                        /path/to/eventlog2
-```
-
-### Output
-By default this outputs a log file under sub-directory `./rapids_4_spark_profile` named
-`rapids_4_spark_tools_output.log` that contains the processed applications. The output will go into your
-default filesystem, it supports local filesystem or HDFS. There are separate files that are generated
-under the same sub-directory when using the options to generate query visualizations or printing the SQL plans.
-
-The output location can be changed using the `--output-directory` option. Default is current directory.
-
-There is a 100 characters limit for each output column. If the result of the column exceeds this limit, it is suffixed with ... for that column.
-
-ResourceProfile ids are parsed for the event logs that are from Spark 3.1 or later. ResourceProfileId column is added in the output table for such event logs. 
-A ResourceProfile allows the user to specify executor and task requirements for an RDD that will get applied during a stage. This allows the user to change the resource requirements between stages.
-  
-Note: We suggest you also save the output of the `spark-submit` or `spark-shell` to a log file for troubleshooting.
-
-Run `--help` for more information.
+Please refer to [this documentation](../docs/additional-functionality/qualification-profiling-tools.md) 
+for more details on how to use the tools.
```

Comment:
If a user lands in this folder from within the github repository, I think it is still useful to give the pre-amble of what these tools are. 

```suggestion
The qualification tool analyzes applications to determine if the RAPIDS Accelerator for Apache Spark 
might be a good fit for those applications.

The profiling tool generates information which can be used for debugging and profiling applications.
The information contains the Spark version, executor details, properties, etc. This runs on either CPU or
GPU generated event logs.

Please refer to [this documentation](../docs/additional-functionality/qualification-profiling-tools.md) 
for more details on how to use the tools.
```

Reply:
@sameerz I think in this tools README, we do have some pre-amble on the top right?

Reply:
Oh yes, I missed that at the top of the file.  I am good with the change.